app/controllers/profile_controller.rb

```diff
@@ -0,0 +1,44 @@
+class ProfileController < PeopleController
```

Comment:
Proszę przywrócić nazwę `ProfilesController` oraz nazwę pliku `profiles_controller.rb` - to był mój błąd.